charts/operator/testdata/kustomization.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+  # TODO: Move these links back to main and then to the tag in appVersion.
+  - https://raw.githubusercontent.com/redpanda-data/redpanda-operator/chris/p/correct-rbac/operator/config/rbac/leader-election-role/role.yaml
```

Comment:
I'm assuming this will get updated after https://github.com/redpanda-data/redpanda-operator/pull/301 merges